docs/tutorial.md

```diff
@@ -237,6 +237,7 @@ kubectl create secret docker-registry regcred \
                     --docker-password=<your-pword> \
                     --docker-email=<your-email>
 ```
+(If Docker Hub is used, then arg docker-server must be set to 'index.docker.io')
```

Comment:
small nit, we tend to put these caveats after a `**Note:**` to ensure users don't miss them, as shown in this snippet from another part of this tutorial: 
https://github.com/tektoncd/pipeline/blob/b7e25054c9e07d44004f20e5618b4ffe354b4b8f/docs/tutorial.md#L14